DataFormats/CaloRecHit/interface/CaloCluster.h

```diff
@@ -25,6 +25,9 @@
 
 namespace reco {
 
+  class CaloCluster;
+  std::ostream& operator<<(std::ostream& out, 
```

Comment:
It is `reco::operator<<` there. But the way C++ looks up free functions, it will look for it in the same namespace as `CaloCluster`.